theme-settings.php

```diff
@@ -184,7 +164,7 @@ function megatron_form_system_theme_settings_alter(&$form, &$form_state) {
   $form['clf_navigation_option']['clf_use_primary_menu_in_drawer'] = array(
     '#type' => 'checkbox',
     '#title' => t('Use the primary menu in the off-canvas drawer?'),
-    '#description' => t('This is optional in case you want to use additonal content blocks, such as a menu block, in the off-canvas drawer region.'),
+    '#description' => t('If not chosen, you should use a menu block or alternate method for main navigation in the off-canvas drawer region.'),
```

Comment:
I agree that the wording could use clarification, I have a hang-up on "chosen" but otherwise +1

Reply:
I changed it to "If you do not use the primary menu in the drawer, you should use a menu block or alternate method for main navigation in the off-canvas drawer region." Hope this is better.